src/view/configuration/sandboxSelector.jsx

```diff
@@ -0,0 +1,96 @@
+/*
+Copyright 2021 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import React, { useEffect } from "react";
+import { useAsyncList } from "@react-stately/data";
+import { Item } from "@adobe/react-spectrum";
+import PropTypes from "prop-types";
+import useIsFirstRender from "../utils/useIsFirstRender";
+import FormikPicker from "../components/formikReactSpectrum3/formikPicker";
+
+const getKey = sandbox => sandbox && sandbox.name;
+const getLabel = sandbox => {
+  if (!sandbox) {
+    return undefined;
+  }
+  const region = sandbox.region ? ` (${sandbox.region.toUpperCase()})` : "";
+  return `${sandbox.type.toUpperCase()}  ${sandbox.title}${region}`;
+};
+
+const SandboxSelector = ({
+  name,
+  defaultSelectedSandbox,
+  onSelectionChange,
+  items,
+  otherProps
+}) => {
+  const isFirstRender = useIsFirstRender();
+
+  const sandboxList = useAsyncList({
```

Comment:
Since `load()` is just returning `{ items }` each time, is there any reason that this cannot be a regular, synchronous list with `useListData()`?

Reply:
yes,  I'll add this into my next PR, good catch!